app/dts/behaviors/sleep.dtsi

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2020 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+/ {
+	behaviors {
+		sleep: behavior_sleep {
```

Comment:
Please add the `/omit-if-no-ref/` here like other behaviors were updated to include.

Reply:
Added